packages/web-components/src/appointment-planner/client/components/appointment-planner.svelte

```diff
@@ -0,0 +1,61 @@
+<script>
+  import { weekStore } from '../core/store/week-store'
+  import Fa from 'svelte-fa'
+  import { faChevronRight, faChevronLeft } from '@fortawesome/free-solid-svg-icons'
+  import * as commonStyles from '../../../common/styles/index'
+  import cx from 'classnames'
+
+  import DateTimePicker from './date-time-picker.svelte'
+
+  import { apointmentPlannerSvg } from '../styles/appointment-planner.styles'
+  import {injectGlobalCss} from '../core/theme'
+
+  export let theme = {}
+  injectGlobalCss(theme)
+</script>
+
+<style type="text/scss">
+
+  .appointment-planner__container {
+    padding: 20px;
+    box-shadow: 1px 1px 7px 1px #d4d4d4;
+    border-radius: 4px;
+    overflow: scroll;
+  }
+
+  .appointment-planner__header-container {
```

Comment:
I don't see what the __ bem is helping with here - we're not using bem - that is all about selector composition and specificity. Can we just stick to `component-name-selector` as a convention.